src/irmin/commit_intf.ml

```diff
@@ -42,6 +42,9 @@ module type S_generic_key = sig
 
   val info : t -> Info.t
   (** The commit info. *)
+
+  val pp : t Fmt.t [@@ocaml.toplevel_printer]
```

Comment:
I added this `pp` with the associated annotation, but it won't actually be used in `utop` as-is since `Store` does not include this signature for its `Commit`. @Ngoguey42 this is what caused me to open #1821 the other day.